### ISSUE TITLE
can't use blend in draw parameters

### ISSUE THREAD

**Issue by LFalch** [tomaka/glium#1229]
When trying to define a `glium::DrawParameters` instance, I get get an error, saying unresolved import. I've tried all sorts of ways of accessing Blend. The code looks something like this:

```rust
use glium::draw_parameters::Blend;

let params = glium::DrawParameters{
    blend: Blend::alpha_blending(),
    .. Default::defualt()
};
```

**Comment by tomaka** [comment]
That's because it was recently changed and the changes are not published on crates.io yet.

In the old API it was `blending_function`: https://github.com/tomaka/glium/blob/1f8068a7b7d55c10a0a83a9cce662978d206a564/src/draw_parameters/mod.rs#L714-L718

**Comment by tomaka** [comment]
The change has been published in glium 0.9.0 now on crates.io
